app/assets/stylesheets/_category.scss

```diff
@@ -24,3 +24,18 @@
   color: #bfa24b;
   text-decoration: none;
 }
+
+#menu .floaters{
+  width:18%;
+  border:1px solid white;
+  @include border-radius(5px);
+  background:rgba(255, 255, 255, 0.83);
+
+  h4{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space